lib/endpoints/class-wp-rest-posts-controller.php

```diff
@@ -824,9 +824,10 @@ protected function prepare_item_for_database( $request ) {
 		// Post title.
 		if ( ! empty( $schema['properties']['title'] ) && isset( $request['title'] ) ) {
 			if ( is_string( $request['title'] ) ) {
-				$prepared_post->post_title = wp_filter_post_kses( $request['title'] );
+				// wp_filter_kses slash handling is lossy: use the underlying methods directly
+				$prepared_post->post_title = wp_kses( $request['title'], current_filter() );
```

Comment:
I thought `current_filter()` here should be `title_save_pre` because post titles get a stricter set of HTML tags allowed than `wp_kses_post()`